dprint.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://dprint.dev/schemas/v0.json",
+  "projectType": "openSource",
+  "incremental": true,
+  "markdown": {
+  },
+  "rustfmt": {
+    "merge_imports": true
```

Comment:
This has been deprecated already, I can recommend you update to the rust plugin `0.4.0`.

Probably best to re-initialize the repo from scratch, as that will pull the latest plugin versions in.

Reply:
It is superseded by `import_granularity`.

Reply:
I pushed a commit that changes that and re-formats - is that good enough? Why do I have to re-initialize the repo?